### PR TITLE
Chore/deku formatting

### PR DESCRIPTION
**Pull request by Gau-thier**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -27,23 +27,23 @@
...
               buildInputs = with pkgs;
                 with ocamlPackages_dev; [
                   ocaml-lsp
-                  ocamlformat
+                  ocamlformat_0_20_1
                   odoc
                   ocaml
                   dune_3
-                  nixfmt
+                  treefmt
```
Comment (ulrikstrid) [inline flake.nix:43]: You probably still want `nixfmt` since `treefmt` will call out to it

File: nix/formatter.nix [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+pkgs.writeShellApplication {
+  name = "format.sh";
+  runtimeInputs = with pkgs; [
+    treefmt
+    nixfmt
+    ocamlformat_0_20_1
+    ocaml
+    dune_3
+    nodePackages.prettier
```
Comment (ulrikstrid) [inline nix/formatter.nix]: Prettier can be removed

**Review by ulrikstrid — CHANGES_REQUESTED** [review]
Added some comments about the dependencies, but looks good otherwise

File: flake.nix (hunk 1 above)
Comment (Gau-thier) [reply flake.nix:43]: My bad, done.

File: nix/formatter.nix (hunk 2 above)
Comment (Gau-thier) [reply nix/formatter.nix]: Thanks, forgot to remove it. Done.

**Review by ulrikstrid — APPROVED** [review]
Looks good

**Review by rosalogia — APPROVED** [review]

**Pull request merged**